From 2e4918a471bda67913b654a01620809b348f0dac Mon Sep 17 00:00:00 2001
From: SeriniTY320 <SeriniTY320@gmail.com>
Date: Thu, 21 Sep 2017 22:13:21 +0000
Subject: [PATCH] Remove-CAP_SYS

---
 fs_config.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/fs_config.c b/fs_config.c
index 5b9d174..d5dd3c4 100644
--- a/fs_config.c
+++ b/fs_config.c
@@ -191,10 +191,10 @@ static const struct fs_path_config android_files[] = {
                                            CAP_MASK_LONG(CAP_SETGID),
                                               "system/bin/run-as" },
 
-    /* Support FIFO scheduling mode in SurfaceFlinger. */
+    /* Revert for MT6735 (Oreo) by SeriniTY 
     { 00755, AID_SYSTEM,    AID_GRAPHICS,  CAP_MASK_LONG(CAP_SYS_NICE),
                                               "system/bin/surfaceflinger" },
-
+    */
     /* Support hostapd administering a network interface. */
     { 00755, AID_WIFI,      AID_WIFI,      CAP_MASK_LONG(CAP_NET_ADMIN) |
                                            CAP_MASK_LONG(CAP_NET_RAW),
-- 
2.7.4

