From d1105c3e606764649f663f9a103950dc48fbcef3 Mon Sep 17 00:00:00 2001
From: SeriniTY320 <SeriniTY320@gmail.com>
Date: Thu, 21 Sep 2017 22:48:33 +0000
Subject: [PATCH] wvm-access patch for oreo

---
 MediaSource.cpp | 21 ++++++++++++++++++++-
 1 file changed, 20 insertions(+), 1 deletion(-)

diff --git a/MediaSource.cpp b/MediaSource.cpp
index bc0206f..180e2dc 100644
--- a/MediaSource.cpp
+++ b/MediaSource.cpp
@@ -22,7 +22,26 @@ MediaSource::MediaSource() {}
 
 MediaSource::~MediaSource() {}
 
-}  // namespace android
+}  // namespace androidz
+
+extern "C" {
+
+bool _ZNK7android11MediaSource11ReadOptions14getNonBlockingEv(android::IMediaSource::ReadOptions *readOptions) {
+    bool res = readOptions->getNonBlocking();
+    return res;
+}
+
+bool _ZNK7android11MediaSource11ReadOptions9getSeekToEPxPNS1_8SeekModeE(android::IMediaSource::ReadOptions *readOptions, int64_t *time_us, android::IMediaSource::ReadOptions::SeekMode *mode) {
+    bool res = readOptions->getSeekTo(time_us, mode);
+    return res;
+}
+
+int64_t _ZNK7android11MediaSource11ReadOptions9getLateByEv(android::IMediaSource::ReadOptions *readOptions) {
+    int64_t res = readOptions->getLateBy();
+    return res;
+}
+
+}
 
 extern "C" {
 
-- 
2.7.4

